src/wagtail_live/models.py

```diff
@@ -171,8 +161,8 @@ def delete_live_post(self, live_post_id):
         if live_post_index == -1:
             return
         del self.live_posts[live_post_index]
-        self.last_update_at = now()
-        self.save()
+
+        self.save_revision()
```

Comment:
Can we add some proof that this has the desired effect? In the form of a test?

This way we 'lockdown' our choices. Future changes in the code, might not break the approach we chose.

Reply:
Maybe make the adding factories PR first, and then revisit this.

Reply:
Yes, I agree with you. We can revisit this after adding some tests.